.github/workflows/main.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     - run: git fetch origin master
     - uses: flux-framework/pr-validator@master
 
-
-  run-checks:
+  python-format:
+    name: python format
```

Comment:
Up to you, but since you are only using a single python version in this check, you could remove the use 
 of a `matrix` here to simplify things. If you do, then remember to update the name of the status check in mergify since it would no longer contain the python version.

Reply:
I agree with you, I can just remove `matrix`. I just pushed up a change that should remove the use of `matrix` and change the name of the check in mergify to just `python format`.